src/mam/mod_mam_utils.erl

```diff
@@ -172,6 +174,11 @@ generate_message_id() ->
     UniqueStamp = mongoose_mam_id:next_unique(CandidateStamp),
     encode_compact_uuid(UniqueStamp, NodeId).
 
+-spec generate_message_id(integer()) -> integer().
```

Comment:
Is the version without arguments still used?

Reply:
To be precise, I don't think so, but I wasn't sure if removing it or not 🤔